ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -74,6 +74,22 @@
             }
         });
     };
+    Zenoss.Dashboard.eventRenderer = function(value, metaData, record) {
+        console.log(record);
+        var table = Zenoss.render.events(value),
```

Comment:
it will be more efficient to check the table and return it if it is empty before running the next piece of code